compile.ps1

```diff
@@ -0,0 +1,5 @@
+#Prerequisite to be ran in a shell with VS2017 dev tools environment loaded
+$url = "https://dist.nuget.org/win-x86-commandline/v4.5.1/nuget.exe"
+Start-BitsTransfer -Source $url -Destination nuget.exe
```

Comment:
Start-BitsTransfer doesnt work via PS remoting, can we replace with the dreaded Invoke-WebRequest?

Reply:
It's meant to be more as a sample script and as a source of inspiration. I don't think it should be used as is.
I.e. the user should have a variable per nuget version, maybe check the SHA checksum to ensure that is the intended file etc.
But sure I can change it to Invoke-WebRequest.